code/modules/martial_arts/combos/adv_first_aid/mend_bleeding.dm

```diff
@@ -0,0 +1,21 @@
+/datum/martial_combo/adv_first_aid/mend_bleeding
+	name = "Mend Artery"
+	steps = list(MARTIAL_COMBO_STEP_HELP, MARTIAL_COMBO_STEP_DISARM, MARTIAL_COMBO_STEP_GRAB, MARTIAL_COMBO_STEP_HARM)
+	explaination_text = "Mends an instance of internal bleeding in the targeted limb."
+
+/datum/martial_combo/adv_first_aid/mend_bleeding/perform_combo(mob/living/carbon/human/user, mob/living/target, datum/martial_art/MA)
+	if(!do_after(user, 10 SECONDS, TRUE, target))
+		to_chat(user, "<span class='warning' You were interrupted! </span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='warning'>You were interrupted!</span>")
```

Didn't close the opening tag properly. You also don't need spaces before/after `>`/`<`

Reply:
Sorry about that, kind of missed that, will get on it real quick